setup.py

```diff
@@ -36,6 +38,25 @@
     version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]", init_file.read(), re.M)
     version_string = version_match.group(1)
 
+try:
+    # Check platform and remove unsupported subtensor node api binaries.
+    if platform == "linux" or platform == "linux2":
+        # linux
+        # remove macos binaries
+        os.remove('bin/subtensor-node-api-macos')
+    elif platform == "darwin":
+        # OS X
+        # remove linux binaries
+        os.remove('bin/subtensor-node-api-linux')
+    else:
+        # neither linux or macos
+        # remove both binaries
+        os.remove('bin/subtensor-node-api-linux')
+        os.remove('bin/subtensor-node-api-macos')
+except Exception as e:
+    print('Failed to remove unsupported subtensor node api binaries. Error: {}'.format(e))
+    pass
```

Comment:
Why do we need to remove binaries?

I agree with the intention of providing just one binary. What about just packaging one binary conditionally and not removing the unused? Would that be possible?

Reply:
The only way I see this being possible is if we download the binary on-demand. The idea here was to conserve space because they're 50MB. 
Not sure users can build the binaries themselves as they require node/npm, might be inconvenient.

Reply:
I see your point.

About this approach, does this means that we are going to maintain one pip package for each platform now?

Reply:
Iirc, the way it works is we have one wheel per platform under the same package. pip should automatically request and install the correct wheel.

Reply:
ah i see, perfect. Thanks @camfairchild